package.json

```diff
@@ -11,11 +11,26 @@
   },
   "dependencies": {
     "next": "12.0.10",
+    "npm": "^8.4.1",
     "react": "17.0.2",
     "react-dom": "17.0.2"
   },
   "devDependencies": {
     "@types/react": "^17.0.39",
+    "@typescript-eslint/eslint-plugin": "^5.10.2",
```

Comment:
As mentioned earlier, Next.js v11.x comes pre-packaged with amazing support for ESLint out-of-the-box! We really don't need the additional dev-dependencies other than: `eslint`, `eslint-config-next` & `prettier` (although we might require some Prettier plugins). So, could you remove the unnecessary dependencies?

That said, here's the change you need to make to the `package.json` file:

```diff
  "devDependencies": {
    "@types/react": "^17.0.39",
-   "@typescript-eslint/eslint-plugin": "^5.10.2",
-   "@typescript-eslint/parser": "^5.10.2",
    "eslint": "^7.23.0",
-   "eslint-config-airbnb": "^19.0.4",
    "eslint-config-next": "12.0.2",
-   "eslint-config-prettier": "^8.3.0",
-   "eslint-import-resolver-typescript": "^2.5.0",
-   "eslint-plugin-import": "^2.25.4",
-   "eslint-plugin-jsx-a11y": "^6.5.1",
-   "eslint-plugin-prettier": "^4.0.0",
-   "eslint-plugin-react": "^7.28.0",
-   "eslint-plugin-react-hooks": "^4.3.0",
-   "@next/eslint-plugin-next": "^12.0.10",
    "prettier": "^2.4.1",
    "typescript": "^4.5.5"
  }
```

Reply:
why do we need eslint if it comes by default with next?

Reply:
Next.js provides some "default configurations" for ESLint which gets depending on the contents of the `.eslintrc.json`. I really suggest taking a look at the [official docs](https://nextjs.org/docs/basic-features/eslint) on how Next.js wraps ESLint to provide a sensible out-of-the-box experience.

You still need to add the `eslint` binary as a dependency, else `npm lint` won't work as expected. The `eslint-config-next` wraps the other ESLint configs & plugins like `eslint-plugin-jsx-a11y`, `eslint-plugin-react` & so on. Please take a look at the docs for more info on it.